### DISCUSSION TITLE
Scroller

### DISCUSSION THREAD

**Discussion by AbhishekCPatil** [community/community#28964] [Other Feature Feedback, Questions, & Ideas]
Dear team, kindly provide a scroller when a README file is opened.
Thank you

**Comment by Akash1134** [comment]
Hi @AbhishekCPatil , We are happy you are here! Would you mind sharing more explicit details about your requirements with Scroller while browsing a ReadMe file?  Giving a few more details might help someone give you a nudge in the right direction. Thanks!
